hover-provider-plugin/src/hover-provider-frontend-plugin.ts

```diff
@@ -0,0 +1,36 @@
+/*********************************************************************
+* Copyright (c) 2018 undefined
```

Comment:
undefined ?

Reply:
Thanks, I'll fix it
I'm preparing a PR for `che-generator-plugin` too.